setup.py

```diff
@@ -4,7 +4,7 @@
     version = version_file.read().strip()
 
 extras_require_serving = [
-    "lightgbm~=2.3",
+    "lightgbm~=3.3.2",
```

Comment:
Should also update the serving `requirements.txt` with pip-comptile
(maybe we can also add a pip-compile step to the gitlab-ci )

Reply:
added an issue to add pip-tools check in the ci